js/src/components/shipping-rate-section/minimum-order-card/changeGroup.test.js

```diff
@@ -1,9 +1,9 @@
 /**
  * Internal dependencies
  */
-import getMinimumOrderChangeHandler from './getMinimumOrderChangeHandler';
+import { changeGroup } from './changeGroup';
 
-describe( 'getMinimumOrderChangeHandler( value, onChange, oldGroup )( newGroup )', () => {
+describe( 'changeGroup( value, oldGroup, newGroup )', () => {
```

Comment:
💅  Not sure about this practice of adding the params in the test describe. That means on each change in the function signature we should remember to change them here as well since the test wont fail if the description is not accurate.

Reply:
I just left it as it was to minimize the number of changes, but I agree with your point. Then given, that the end goal is to merge it to 1410, which adds it from scratch, I'd remove them.